completion/zsh/_tlp-rdw

```diff
@@ -0,0 +1,5 @@
+#compdef tlp-rdw
+
+_values 'command' \
+    'disable[disable RDW actions]' \
+    'enable[enable RDW actions]'
```

Comment:
When I enter (or complete to) `tlp-rdw enable ` (note the trailing space) the completion will continue and happily add any number of _enable_ / _disable_ arguments. There should only be one. Also happens for _disable_ as 1st argument.

Reply:
I learnt something new about how zsh completion works today! Thanks